### DISCUSSION TITLE
ValueError: You are using a deprecated configuration of Chroma.

### DISCUSSION THREAD

**Discussion by Umarasma** [imartinez/privateGPT#947] [Q&A]
Did anyone updated Chroma Client to "New Clients" ?

Thanks & Regards

Error:
If you do not have data you wish to migrate, you only need to change how you construct
your Chroma client. Please see the "New Clients" section of https://docs.trychroma.com/migration.
________________________________________________________________________________________________

If you do have data you wish to migrate, we have a migration tool you can use in order to
migrate your data to the new Chroma architecture.
Please `pip install chroma-migrate` and run `chroma-migrate` to migrate your data and then
change how you construct your Chroma client.

See https://docs.trychroma.com/migration for more information or join our discord at https://discord.gg/8g5FESbj for help!

**Comment by Tash-Feneko** [comment]
I have the same issue. Tried installing newer chroma version, did not help.

**Reply by blackhawk23x** [reply]
Same issue here. Tried adding the new client but it still fails. RIP

**Reply by Tash-Feneko** [reply]
I actually managed to override this issue. The script is prepared for older version of Chroma, so it will work if you downgrade to 0.3.29.
Try `pip uninstall chromadb && pip install chromadb==0.3.29`

**Reply by blackhawk23x** [reply]
That worked for me thanks.
Edit: I had issues importing the requirements so I added them in manually.
